.pylintrc

```diff
@@ -460,6 +453,8 @@ init-import=no
 # builtins.
 redefining-builtins-modules=six.moves,past.builtins,future.builtins
 
+disable=W0640
```

Comment:
I haven't seen the need to disable this one before, but it looks like a potentially [dangerous situation](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/cell-var-from-loop.html) so better if we can keep it!